### ISSUE TITLE
Documents folder won't display (web site)

### ISSUE THREAD

**Issue by g33kdad** [nextcloud/server#11908]
Good Morning,

When I click on my "Documents" folder, i get a spinning circle but the list of files never loads. All of the other folders on my server will load and i can access their contents, however the folder labled "Documents" will not load.

 
### Steps to reproduce
1. log into my server
2. click on "Documents" folder (https://cloud._my-web-site_.net/index.php/apps/files?dir=//Documents)
3.

### Expected behaviour
List of files and folders in "Documents" should appear on the screen

### Actual behaviour
the page loads but in the pane were the content should be, i just get the spinning circle indicating it's "thinking" and the contents of the directory never show on the screen.

### Server configuration

**Operating system**:
linux
**Web server:**
apache2
**Database:**
mysql
**PHP version:**
7.0
**Nextcloud version:** (see Nextcloud admin page)
Nextcloud 14.0.3 (beta channel)
**Updated from an older Nextcloud/ownCloud or fresh install:**
updated
**Where did you install Nextcloud from:**
don't remember
**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.

```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.0.1
  - activity: 2.7.0
  - admin_audit: 1.4.0
  - audioplayer: 2.4.1
  - bookmarks: 0.13.1
  - bruteforcesettings: 1.1.0
  - calendar: 1.6.3
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - contacts: 2.1.6
  - dav: 1.6.0
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_external: 1.5.0
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.3.2
  - files_retention: 1.3.0
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
  - gallery: 18.1.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - music: 0.9.2
  - nextcloud_announcements: 1.3.0
  - notes: 2.4.2
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - password_policy: 1.4.0
  - provisioning_api: 1.4.0
  - qownnotesapi: 18.8.0
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - spreed: 4.0.0
  - support: 1.0.0
  - survey_client: 1.2.0
  - systemtags: 1.4.0
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.1
  - unsplash: 1.1.1
  - updatenotification: 1.4.1
  - user_external: 0.4
  - user_ldap: 1.4.0
  - workflowengine: 1.4.0
Disabled:
  - encryption
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "10.46.0.5",
            "nextcloud",
            "cloud.thestrangeland.net"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/cloud.thestrangeland.net\/",
        "dbtype": "mysql",
        "version": "14.0.3.0",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "appstore.experimental.enabled": "true",
        "mail_smtpmode": "smtp",
        "mail_smtpauth": 1,
        "mail_smtpport": "465",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "filelocking.enabled": true,
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "dbindex": 0
        },
        "htaccess.RewriteBase": "\/",
        "maintenance": false,
        "loglevel": 2,
        "theme": "",
        "updater.release.channel": "beta",
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
r
or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
yes / local
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no
#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**
firefox
**Operating system:**
linux
#### Web server error log
<details>
<summary>Web server error log</summary>

```

```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```

```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by sbuller** [comment]
I'm seeing something very much like this. That is to say, a folder which when I try to list it it simply shows a spinning icon and never displays its contents.

There's one particular folder that's not showing its contents, and shares of said folder between users also cannot be viewed on the site. The folder can be downloaded as a zip, and the files are intact on the server. In the JS console I get an error message when displaying that folder that I don't get on others; it says "URIError: malformed URI sequence       client.js:283:11". Webdav access works fine.

There's spaces in the folder's name, as well as those of its children. One of its children also has an ampersand in its name. That said, an attempt to replicate the problem folder is so far unsuccessful. I'll keep trying.

**Comment by sbuller** [comment]
```
$ cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```
```
$ ./occ status
  - installed: true
  - version: 14.0.3.0
  - versionstring: 14.0.3
  - edition:
```

And the full browser console on the affected page:
```
Content Security Policy: Directive ‘frame-src’ has been deprecated. Please use directive ‘child-src’ instead.  (unknown)
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src 'nonce-YldGNFBrT3hCTlA4RDlYMmVVZjRxTFFidVNRYUxIWmdFTWthT3YrMFc5WT06SWdJZlpuYlFSNzZ5VmJldktqMmVuSTFCMjNSSVlnTWtmNWt4YzdiMkkrUT0=' 'unsafe-eval'”). Source: ;!function(){var t,e,n,o=0,u=function(t,....  files:1
JQMIGRATE: Migrate is installed, version 1.4.0  core.js:7:542
window.controllers/Controllers is deprecated. Do not use it for UA detection.  merged.js:2176
New search handler registered  search.js:66:4
querying Whats New data was successful: nocontent  merged-template-prepend.js:4480:4
undefined  merged-template-prepend.js:4481:4
URIError: malformed URI sequence[Learn More]  client.js:283:11
```

**Comment by sbuller** [comment]
My bad. I said 'ampersand', and was even testing ampersands. The trouble character is a '%'. It seems that `_parseFileInfo` in client.js calls `decodeURIcomponent` on the URI twice.

**Comment by sbuller** [comment]
Well, looks like this is already addressed by commit 0f030d885b735fa52536578d2cd269e3e9e39ffa from 13 days ago.

**Comment by MorrisJobke** [comment]
Fixed by #11744 and will be shipped in 14.0.4
